packages/api/tests/endpoints/test-rules.js

```diff
@@ -231,6 +236,14 @@ test('CUMULUS-911 POST with pathParameters and without an Authorization header r
   assertions.isAuthorizationMissingResponse(t, response);
 });
 
+test('CUMULUS-911 PATCH with pathParameters and without an Authorization header returns an Authorization Missing response', async (t) => {
```

Comment:
What does `CUMULUS-911` mean?

Reply:
No idea.  But filled the 'todo' tests. [5f8816f820](https://github.com/nasa/cumulus/pull/3472/commits/5f8816f820c415d1aed5b0d043dd4fe9adb7e929)